ci-operator/step-registry/openshift/e2e/cert-rotation/suspend/sno/openshift-e2e-cert-rotation-suspend-sno-workflow.yaml

```diff
@@ -3,23 +3,30 @@ workflow:
   steps:
     allow_skip_on_success: true
     allow_best_effort_post_steps: true
-    cluster_profile: packet-sno
+    cluster_profile: packet-assisted
+    env:
+      PACKET_PLAN: "c3.medium.x86"
+      PACKET_OS: "rocky_8"
+      ASSISTED_CONFIG: |
+        KUBE_API=true
+        ENABLE_KUBE_API=true
+        ISO_IMAGE_TYPE=full-iso
+        IPv4=true
+        TEST_FUNC=test_kubeapi
+        MAKEFILE_TARGET="test_kube_api_parallel"
+        DISCONNECTED=true
+        NUM_MASTERS=1
     pre:
       - ref: baremetalds-packet-setup
-      - ref: ipi-install-rbac
-      - ref: ipi-install-hosted-loki
-      - ref: baremetalds-sno-setup
+      - chain: openshift-e2e-cert-rotation-conf
```

Comment:
`ref: baremetald-packet-setup` used hardcode IP for SNO, there may be conflicts with other tests.
detail see: https://github.com/openshift/release/blob/master/ci-operator/step-registry/baremetalds/sno/setup/baremetalds-sno-setup-commands.sh#L68-L71

Reply:
Right, we should not be using `baremetald` but assisted flow (it runs minikube so its easier to create mirror there)